Signal/src/call/PeerConnectionClient.swift

```diff
@@ -110,10 +115,10 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
 
     private var videoSender: RTCRtpSender?
     private var localVideoTrack: RTCVideoTrack?
-    private var remoteVideoTrack: RTCVideoTrack?
+    private weak var remoteVideoTrack: RTCVideoTrack?
```

Comment:
I spent a lot of time wrestling with exceptions in the destructor of RTCVideoTrack.  RTCVideoTrack is apparently very fragile and will throw exceptions in its destructor if you retain a strong reference to it _anywhere_.